src/filters/getFullServiceName.js

```diff
@@ -1,7 +1,10 @@
 import config from '../../config/index.js'
 
-export default (service) => {
-  const serviceName = config.serviceName
-
-  return serviceName.replace('Provide', service)
+const getFullServiceName = (service) => {
+  if (!service || typeof service !== 'string') {
+    throw new Error('Service name must be a non-empty string')
+  }
+  return config.serviceName.replace('Provide', service)
 }
+
+export default getFullServiceName
```

Comment:
this is so we can correctly name each of the sub services.

i.e. 

provide planning and housing data
submit planning and housing data
manage ...
etc